requirements/deploy.txt

```diff
@@ -205,21 +205,21 @@ elasticsearch-dsl==8.13.0
     # via
     #   -r requirements/pip.txt
     #   django-elasticsearch-dsl
-exceptiongroup==1.2.0
+exceptiongroup==1.2.1
     # via ipython
 executing==2.0.1
     # via stack-data
-filelock==3.13.3
+filelock==3.13.4
     # via
     #   -r requirements/pip.txt
     #   virtualenv
 funcy==2.0
     # via
     #   -r requirements/pip.txt
     #   django-cacheops
-gunicorn==21.2.0
+gunicorn==22.0.0
```

Comment:
I think the breaking changes don't affect us: https://docs.gunicorn.org/en/stable/news.html#id1